src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+
+import classNames from 'classnames';
+
+type Props = {
+  errorMessage: string
+};
+
+export const Notification: React.FC<Props> = ({ errorMessage }) => {
+  const [isHidden, setIsHidden] = useState(false);
+
+  const autoHide = () => {
+    setTimeout(() => {
+      setIsHidden(true);
+    }, 3000);
+  };
+
+  autoHide();
+
+  return (
+    <div
+      className={classNames(
+        'notification',
+        'is-danger',
+        'is-light',
+        'has-text-weight-normal',
+        { hidden: isHidden },
+      )}
+    >
+      <button // eslint-disable-line jsx-a11y/control-has-associated-label
```

Comment:
its not a comment, I wrote that's cause I can't write `<button></button>` because it automatically back to `<button/>`
that's why linter throw `warning : jsx-a11y/control-has-associated-label`, so, and I put `eslint-disable-line jsx-a11y/control-has-associated-label` to disable this warning)), Thank you